.github/workflows/test.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: ['3.9', '3.10', '3.11', '3.12', '3.13']
+        python-version: ['3.10', '3.11', '3.12', '3.13', '3.14.0-rc.2']
```

Comment:
Sorry for being pedantic but I don't see why the latest release candidate should be added in the same commit that removes 3.9.